cylc/flow/pathutil.py

```diff
@@ -260,24 +261,24 @@ def remove_dir_and_target(path: Union[Path, str]) -> None:
     Args:
         path: the absolute path of the directory to delete.
     """
+    log = LOG.info if cylc.flow.flags.verbosity == 1 else LOG.debug
```

Comment:
I think these should stay at `LOG.debug`, the visibility of the messages being decided by the level of the log handlers.

Is there a reason that approach won't work here?

Reply:
Ideally there would be a log level between `INFO` and `DEBUG` for the `-v` level, I suppose. Or are you saying we can manually set the `CYLC_LOG` handler level to `DEBUG` for `cylc clean -v`, inside the `init_clean` function?

Reply:
Ok, got it, DEBUG messages don't show in VERBOSE mode so you made them INFO level to make them visible.

VERBOSE mode seems a bit pointless ATM, I'm trying to remember what the original idea was...

Options:

1. We could create a custom log level for VERBOSE (`logging` supports this). 
   * However, do we really need it? Would it really be helpful to separate log messages like this?
   * (related) We have talked about creating a TASK log level (at ~ the same level as info) to help separate task and scheduler events in the log.
2. We could set the log level to DEBUG in verbose mode
   * I've forgotten what the original idea was, but this would make sense.

With (2) the levels would become:

> QUIET

Shows WARNING or higher.

> INFO

Shows INFO or higher.

> VERBOSE

Shows DEBUG or higher.

> DEBUG

Shows DEBUG or higher with full traceback for debug purposes.

Reply:
I've decided to make the `cylc.flow.pathutil` functions for deleting files/dirs/symlinks log at `INFO` level so that it's clear what's being deleted. This undoes 6066648b9d1390dd1da386ec7630410ce852e3a6. I hope this is fair - it only logs the bare minimum of deleted things (if it's removing a directory, it only logs that, not any constituent files; if a glob matches children of a matching parent, it only logs the parent etc). The only places where the logging may get a bit much are when   the workflow has run on many different install targets, or when doing `--rm 'log/job/*'` instead of `--rm log/job` for a workflow with many cycle points.